content/tutorials/jwt-authentication.textile

```diff
@@ -54,123 +78,156 @@ h2(#live-demo). Ably JWT auth example
+    jwt.sign(jwtPayload, apiSecret, jwtOptions, (err, tokenId) => {                                                                                                                                                                                            
+        console.log('JSON Web Token signed by auth server')                                                                                                                                                                                                    
+        if (err) {                                                                                                                                                                                                                                             
+            console.trace()                                                                                                                                                                                                                                    
+            return                                                                                                                                                                                                                                             
+        }                                                                                                                                                                                                                                                      
+        res.header('Cache-Control', 'private, no-cache, no-store, must-revalidate')                                                                                                                                                                            
+        res.setHeader('Content-Type', 'application/json')                                                                                                                                                                                                      
+        console.log('Sending signed JWT token back to client')                                                                                                                                                                                                 
+        res.send(JSON.stringify(tokenId));                                                                                                                                                                                                                     
+    })                                                                                                                                                                                                                                                         
```

Comment:
You should trim all the trailing whitespace from this code:

```suggestion
app.get('/auth', (req, res) => {
    console.log('Sucessfully connected to the server auth endpoint')
    jwt.sign(jwtPayload, apiSecret, jwtOptions, (err, tokenId) => {
        console.log('JSON Web Token signed by auth server')
        if (err) {
            console.trace()
            return
        }
        res.header('Cache-Control', 'private, no-cache, no-store, must-revalidate')
        res.setHeader('Content-Type', 'application/json')
        console.log('Sending signed JWT token back to client')
        res.send(JSON.stringify(tokenId));
    })
                                                                                                                                                                                                                                                
```

Reply:
oh that's weird!